### ISSUE TITLE
Clarify disabled edit state

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#2422]
**Description:**

When a breakpoint has >100 hits we do not let the user edit it, but that is not obvious. We should provide a better tooltip and disabled state.

**Comment by jonbell-lot23** [comment]
@jasonLaster do you have a Replay handy with this error so I can have a handy URL to use?

**Comment by jonbell-lot23** [comment]
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/9154902/120249338-a3ace280-c2ce-11eb-8488-a248c40bf050.png">

I went looking into this issue and I found this line that was called more than 100 times, but I think I'm able to edit it. (although it is slowing my browser down and throwing errors)

So it sounds like we need to do this:

1. Confirm that we want to cap this at 100, and make sure that code is working.
2. Get a repro case for this, ideally in a Replay everyone can see.
3. Provide a new design
